packages/shared/src/modules/tasks/libs/types/task-create-dto.type.ts

```diff
@@ -2,7 +2,6 @@ type TaskCreateDto = {
 	categoryId: number;
 	categoryName: string;
 	description: string;
-	dueDate: string;
```

Comment:
Why was it removed?

Reply:
dueDate is added when a TaskDto is created, therefore it is impossible for TaskCreateDto to have a due date